tests/devices/test_taskify_autograd.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
```

Comment:
Should the `qml.taskify` test files be placed one level higher or into a `tests/taskify/` dedicated folder? It seems these are less devices specific and I'm wondering if they belong into `tests/devices/`.

Reply:
Yea, this is a good point. `tests/taskify/` seems like a good option.